spec/requests/loans_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'Loans', type: :request do
+  describe 'POST /loans' do
+   before do
+     @user = create(:user)
+     @creditor = create(:creditor)
+   end
+   
+    let(:loan_params) do
+      {
+        loan: {
+          name: 'Student Loan',
+          balance: 1000,
+          min_payment: 50,
+          interest: 0.05,
+          description: 'My student loan',
+          creditor_id: @creditor.id,
+          user_id: @user.id
+        }
+      }
+    end
+
+    it 'creates a new loan' do
+      expect{ post('/loans', params: loan_params) }.to change { Loan.count }.by(1)
+    end
+  end
```

Comment:
Shall we also add test cases listing, updating and deleting a record?
While adding test cases for these, you may need to create a new factory `:loans`, refer to https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#associations to understand how to define associations on factory bot